recipes/CImg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "CImg" %}
+{% set version = "2.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dtschump/CImg/archive/v.{{ version }}.tar.gz 
+  sha256: 3ad10cb8842e945af208a05b8ed36f4428317ef754c6e341e4ca6ac70b8a8540
+
+build:
+  number: 0
+
+requirements:
+  build:
+  host:
+  run:
+
+test:
+  imports:
```

Comment:
You can remove this line

Reply:
You mean the imports line right? Should I just remove the `requirements` section altogether since it is empty?

Reply:
I just mean `imports` in the test section.